README.md

```diff
@@ -1145,6 +1146,14 @@ For more, see [flake8-print](https://pypi.org/project/flake8-print/) on PyPI.
 | T201 | print-found | `print` found |  |
 | T203 | p-print-found | `pprint` found |  |
 
+### flake8-pyi (Y)
```

Comment:
I think we should consider using `PYI` for this instead of `Y`, which is generic enough that it may well conflict with other plugins.

Reply:
Now we pray that there's no plugin using `PYI` already.

Reply:
I can't find any such references on GitHub Code Search.